config/config.go

```diff
@@ -42,8 +42,8 @@ func DetermineUseLiveData(logger golog.Logger, liveData *bool, sensors []string)
 	return useLiveData, nil
 }
 
-// AttrConfig describes how to configure the SLAM service.
-type AttrConfig struct {
+// Config describes how to configure the SLAM service.
+type Config struct {
```

Comment:
Follows the naming convention Eric recommends https://github.com/viamrobotics/rdk/pull/2225/files#diff-041927d429c5b9c2868ecfc5c12d0aa719ea98978dad15f65ab76f25c5440435R69